tests/integrationTests/tests/c_failure_messages/__init__.py

```diff
@@ -5,12 +5,13 @@
 import os
 import glob
 import shutil
+from submitty_utils import submitty_schema_validator
 
 ############################################################################
 # COPY THE ASSIGNMENT FROM THE SAMPLE ASSIGNMENTS DIRECTORIES
 
-SAMPLE_ASSIGNMENT_CONFIG = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/c_failure_messages/config"
-SAMPLE_SUBMISSIONS       = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/c_failure_messages/submissions"
+SAMPLE_ASSIGNMENT_CONFIG = os.path.join(SUBMITTY_INSTALL_DIR, "more_autograding_examples/c_failure_messages/config")
+SAMPLE_SUBMISSIONS       = os.path.join(SUBMITTY_INSTALL_DIR, "more_autograding_examples/c_failure_messages/submissions")
```

Comment:
If we're going to fix this to use join, let's separate out each part into its own string and passed in and do it right.